client/my-sites/themes/index.node.js

```diff
@@ -12,5 +12,6 @@ export default function( router ) {
 	if ( config.isEnabled( 'manage/themes' ) ) {
 		router( '/design', makeLayout );
 		router( '/design/type/:tier', makeLayout );
+		router( '/design/*', makeLayout ); // Needed so direct hits don't result in a 404.
```

Comment:
Only isomorphic ones -- and since `/theme` is covered, since its server and client side ones are identical, this one's the only one left to cover.